### PR TITLE
Update Team and Business onboarding

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes

- Add Hardened Docker Desktop to Business onboarding.
- Align steps better with UI onboarding guide.

### Related issues (optional)

DA-181
ENGDOCS-1100

### CODE REVIEW

File: docker-hub/onboard-business.md [hunk 1]
```diff
@@ -46,32 +46,37 @@ After completing the steps from the welcome email, verify that your organization
...
 
+## Step 4: Invite members
 
-## Step 4: Enforce sign-in for Docker Desktop
+Add members to your organization using a CSV file, or by entering their email addresses. For more details, see [Invite members](../docker-hub/members.md/#invite-members){: target="_blank" rel="noopener" class="_"}
+
+## Step 5: Manage members with SSO and SCIM
+
+Automate adding members to your organization using the following:
+   - Single Sign-on: Automatically provision and add members when they sign in to Docker through your identity provider. For details, see [Single Sign-on overview](../single-sign-on/index.md).
```
Comment (jerae-duffin) [inline docker-hub/onboard-business.md:62]: Should we say Docker Hub instead of Docker?

Comment (craig-osterhout) [reply docker-hub/onboard-business.md]: Thanks. Updated in the commit below.

File: docker-hub/onboard-business.md [hunk 2]
```diff
@@ -75,8 +75,9 @@ Configure security settings and manage your repositories:
...
-   - [Settings Management](../desktop/hardened-desktop/settings-management/index.md) to control Docker Desktop settings on your members machines.
-   - [Image Access Management](../docker-hub/image-access-management.md/) to control which images your members can access.
-   - [Registry Access Management](../docker-hub/registry-access-management.md/) to control which registries your members can access.
+   - [Settings Management](../desktop/hardened-desktop/settings-management/index.md), which helps you to confidently manage and control the usage of Docker Desktop within your organization.
+   - [Enhanced Container Isolation](../desktop/hardened-desktop/enhanced-container-isolation/index.md), a setting that instantly enhances security by preventing containers from running as root in Docker Desktop’s Linux VM.
+   - [Image Access Management](../docker-hub/image-access-management.md/), which gives you control over which images developers can pull from Docker Hub.
+   - [Registry Access Management](../docker-hub/registry-access-management.md/), which allows you to control the registries developers can access.
```
Comment (jerae-duffin) [inline docker-hub/onboard-business.md:81]:
```suggestion
   - [Registry Access Management](../docker-hub/registry-access-management.md/), lets you control the registries developers can access.
```

File: docker-hub/onboard-business.md [hunk 3]
```diff
@@ -75,8 +75,9 @@ Configure security settings and manage your repositories:
...
 - Configure [Hardened Docker Desktop](../desktop/hardened-desktop/index.md) to improve your organization’s security posture for containerized development. Hardened Docker Desktop includes:
-   - [Settings Management](../desktop/hardened-desktop/settings-management/index.md) to control Docker Desktop settings on your members machines.
-   - [Image Access Management](../docker-hub/image-access-management.md/) to control which images your members can access.
-   - [Registry Access Management](../docker-hub/registry-access-management.md/) to control which registries your members can access.
+   - [Settings Management](../desktop/hardened-desktop/settings-management/index.md), which helps you to confidently manage and control the usage of Docker Desktop within your organization.
+   - [Enhanced Container Isolation](../desktop/hardened-desktop/enhanced-container-isolation/index.md), a setting that instantly enhances security by preventing containers from running as root in Docker Desktop’s Linux VM.
+   - [Image Access Management](../docker-hub/image-access-management.md/), which gives you control over which images developers can pull from Docker Hub.
```
Comment (jerae-duffin) [inline docker-hub/onboard-business.md:80]:
```suggestion
   - [Image Access Management](../docker-hub/image-access-management.md/), lets you control which images developers can pull from Docker Hub.
```

**Review by jerae-duffin — APPROVED** [review]
LGTM!

**Pull request merged**